Assets/Mirror/Editor/Weaver/Program.cs

```diff
@@ -23,15 +23,15 @@ public static void Error(string msg)
 
     public class Program
     {
-        public static bool Process(string unityEngine, string netDLL, string outputDirectory, string[] assemblies, string[] extraAssemblyPaths, IAssemblyResolver assemblyResolver, Action<string> printWarning, Action<string> printError)
+        public static bool Process(string unityEngine, string netDLL, string outputDirectory, string[] assemblies, string[] extraAssemblyPaths, Action<string> printWarning, Action<string> printError)
         {
             CheckDLLPath(unityEngine);
             CheckDLLPath(netDLL);
             CheckOutputDirectory(outputDirectory);
             CheckAssemblies(assemblies);
             Log.WarningMethod = printWarning;
```

Comment:
Why do we receive logging functions?  can we not just call Debug.LogWarning ourselves?  It is not like this code will ever be used outside of unity.

Reply:
we should do this in a separate PR. the functions are needed for weaver tests right now. maybe we can move them into the weaver directly though.